Xamarin.Forms.Core/Layout.cs

```diff
@@ -320,9 +320,7 @@ internal static void LayoutChildIntoBoundingRegion(View child, Rectangle region,
 
 		internal virtual void OnChildMeasureInvalidated(VisualElement child, InvalidationTrigger trigger)
 		{
-			ReadOnlyCollection<Element> children = LogicalChildrenInternal;
-			int count = children.Count;
-			for (var index = 0; index < count; index++)
```

Comment:
There are many cases in the source code where `LogicalChildrenInternal` is being used directly instead of being converted to a local variable. Get count directly from the property.